src/serverrpc.cpp

```diff
@@ -106,7 +106,7 @@ CServerRpc::CServerRpc ( CServer* pServer, CRpcServer* pRpcServer, QObject* pare
     /// @result {string} result.city - The server city.
     /// @result {number} result.countryId - The server country ID (see QLocale::Country).
     /// @result {string} result.welcomeMessage - The server welcome message.
-    /// @result {string} result.directoryServer - The directory server to which this server requested registration, or blank if none.
+    /// @result {string} result.directoryServer - The directory to which this server requested registration, or blank if none.
```

Comment:
Did you run the Python script to generate the md file or did you edit it manually?

Reply:
Oops, this wasn't meant to change.  It's meant to be covered by the "TODO" about the doccomment.

Reply:
That's probably why the check fails

Reply:
It still fails without this line changing.
![image](https://user-images.githubusercontent.com/1549463/227785746-19507778-fdad-4eef-be19-a804cc972488.png)